<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingUp</class>
 <widget class="QMainWindow" name="SingUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wallet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="userNameLineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="confirmPasswordLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerPushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WALLET</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerWasSuccessfulLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Registration was successful and you can</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginRedirectionPushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="incorrectDataLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Please try again incorrect data</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordsNotMatchLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>The passwords do not match</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnFromSignUpButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
